<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogNewDownload</class>
 <widget class="QDialog" name="DialogNewDownload">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>123</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelVideoURL">
     <property name="text">
      <string>Youtube video URL:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="editUserInput"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
     <item>
      <widget class="QRadioButton" name="radioVideoAudio">
       <property name="text">
        <string>Video and Audio</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioAudioOnly">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Only Audio</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="btnboxOkCancel">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnboxOkCancel</sender>
   <signal>accepted()</signal>
   <receiver>DialogNewDownload</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnboxOkCancel</sender>
   <signal>rejected()</signal>
   <receiver>DialogNewDownload</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
